tests/unit/test_init.py

```diff
@@ -0,0 +1,1013 @@
+"""
+SPDX-FileCopyrightText: © Sebastian Thomschke and contributors
```

Comment:
When adding new files with new code please either use your name in the copyright header or use something like "SPDX-FileCopyrightText: © The kleinanzeigen-bot authors and contributors"